.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         sudo apt-get install clang-format
     - run: flake8
     - run: ./scripts/clang-format-diff.sh
+    - run: python ./scripts/check_clean.py
```

Comment:
I think this needs to run after the build, right?  (i.e. it probably needs to move to "build")

Reply:
Good point. Will fix.